hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/WritableTokenRelationStore.java

```diff
@@ -73,12 +75,18 @@ public void commit() {
      * Returns the {@link TokenRelation} with the given token number and account number.
      * If no such token relation exists, returns {@code Optional.empty()}
      *
-     * @param tokenNum - the number of the token to be retrieved
-     * @param accountNum - the number of the account to be retrieved
+     * @param accountId - the number of the account to be retrieved
+     * @param tokenId   - the number of the token to be retrieved
      */
-    public Optional<TokenRelation> getForModify(final long accountNum, final long tokenNum) {
-        final var token =
-                Objects.requireNonNull(tokenRelState).getForModify(EntityNumPair.fromLongs(accountNum, tokenNum));
+    @NonNull
+    public Optional<TokenRelation> getForModify(@NonNull final AccountID accountId, @NonNull final TokenID tokenId) {
+        requireNonNull(accountId);
+        requireNonNull(tokenId);
+
+        if (AccountID.DEFAULT.equals(accountId) || TokenID.DEFAULT.equals(tokenId)) return Optional.empty();
```

Comment:
Do we anticipate `DEFAULT` ids being passed in here so commonly a performance optimization is merited?

Reply:
Originally I had `accountId.accountNumOrThrow()` but wasn't sure what status would be returned if that call threw an NPE. This was the suggested way to deal with that. It wasn't meant to be a performance optimization